### DISCUSSION TITLE
719

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#900] [Comments]
# 719

최근에 교육생분들을 대상으로 신입 개발자를 준비할때 있었던 일들에 대해 이야기를 나누는 시간을 가졌다. 이제막 개발을 시작하게된 분들도 계시고, 컴퓨터 공학을 전공했지만 계속해서 취업에 실패해서 교육기관에 참여하신분들도 계시고 다양했다. 공통된 질문 기저에 깔린 걱정은 "내가 다른 사람에 비해 부족한데 어떻게 극복하느냐" 였다. 그래서 그에 대한 내 생각을 전달드렸다. 국비교육기관이나 부트캠프에서 여러 동기들과 함께 개발을 하거나, 신입 개발자로 입사하여 비슷한 연차의 다른 개발자들과 같이 일을 하다보면 내가 썩 영민하진 못하다는 생각을 하게 된다. 이클립스는 커녕 Java도 처음 하는 신입 개발자분이 나보다 훨씬 더 좋은 퀄리티로 과제를 해결한다던가 중학교부터 코딩을 시작해서 나와 같은 나이에 이미 개..

https://jojoldu.tistory.com/719

**Comment by archan0621** [comment]
향로님 좋은글 감사합니다.
최근에 비슷한 일로 생각이 흩어져 있었는데 덕분에 다시 정신차리고 잡을 수 있게 된것 같습니다.
전 18살에 취업해서 올해로 3년차가 되어가는 풀스택 개발자입니다.
회사에서 배우는걸로 만족하지 않고 항상 퇴근하고 2시간씩 인강을 듣고 사이드 프로젝트를 진행하고 블로그에 글을 올리고 이런 노력으로 당장의 가속력이 뛰어난편은 아니지만 쭉 달리다보면 언젠간 이루리라 생각합니다

**Comment by al-bur** [comment]
공감합니다.
운동이든, 공부든 짧은 시간안에 집중하는 것은 생각보다 쉬운데 그걸 꾸준히 몇 년동안 매일 해나가는 것이 제일 어렵더라고요..
아직까지도 제가 꾸준히 해야될 것들을 어떻게 하면 중간에 힘들어서 포기하지 않고 할 수 있을까를 매번 고민하면서 지내는 데 명확한 해결법은 찾지 못했습니다
